doc/source/packaging.rst

```diff
@@ -3,19 +3,22 @@ Packaging releases
 ==================
```

Comment:
This has a lot of overlap with development/packaging.rst. Do we need both?

Reply:
I noted that as well. Also, when I try and build the documentation I see:
```
/home/rt17603/openghg/doc/source/packaging.rst: WARNING: document isn't included in any toctree
```
so this doesn't seem to be included in the documentation build at the moment anyway.

I think any useful details from this should be incorporated into `development/packaging.rst` and this file should be deleted.

Reply:
done